pkg/filter/stream/dubbo/var.go

```diff
@@ -0,0 +1,20 @@
+package dubbo
+
+import (
+	"mosn.io/mosn/pkg/types"
+	"mosn.io/mosn/pkg/variable"
+)
+
+var (
+	buildinVariables = []variable.Variable{
+		variable.NewIndexedVariable(types.VarDubboRequestService, nil, nil, variable.BasicSetter, 0),
+		variable.NewIndexedVariable(types.VarDubboRequestMethod, nil, nil, variable.BasicSetter, 0),
+	}
+)
+
+func init() {
+	// variable must registry
+	for idx := range buildinVariables {
```

Comment:
you should registry when dubbo filter enable.

Reply:
run example again.

Reply:
Alreay run example succ.